modules/ROOT/pages/dataweave-type-selector.adoc

```diff
@@ -0,0 +1,209 @@
+= Type Selector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The type selector allows the use of the `.` selector over xref:dataweave-type-system.adoc#composite-types[types] which enables you to easily declare types from existing ones. This feature can also be combined with the new xref:dataweave-selectors-reusing-types.adoc[module loader feature] to load files and translate declarations inside your file into DataWeave type directives which can be accessed in the same way as types from any other module:
```

Comment:
```suggestion
Using the type selector (`.`) over xref:dataweave-type-system.adoc#composite-types[types] enables you to declare new types from existing ones. Combined with the DataWeave xref:dataweave-selectors-reusing-types.adoc[module loader], you can also load files and translate declarations inside your file into DataWeave type directives that can be accessed in the same way as types from any other module::
```

Reply:
Suggest avoiding use of terms like "the new" since it won't be new in the next version and "easily" since it's kind of subjective.

Reply:
see if the dev is ok with this formulation

Reply:
I'm concerned about our terminology for this selector also. We should all discuss. We use the dot selector for several similar purposes. Sorry to ramble across several comments... let's discuss with the dev.